spec.html

```diff
@@ -4500,12 +4501,12 @@ <h1>CopyDataBlockBytes ( _toBlock_, _toIndex_, _fromBlock_, _fromIndex_, _count_
         <p>The abstract operation CopyDataBlockBytes takes arguments _toBlock_, _toIndex_, _fromBlock_, _fromIndex_, and _count_. It performs the following steps when called:</p>
         <emu-alg>
           1. Assert: _fromBlock_ and _toBlock_ are distinct Data Block or Shared Data Block values.
-          1. Assert: _fromIndex_, _toIndex_, and _count_ are integer values &ge; 0.
-          1. Let _fromSize_ be the number of bytes in _fromBlock_.
-          1. Assert: _fromIndex_ + _count_ &le; _fromSize_.
-          1. Let _toSize_ be the number of bytes in _toBlock_.
-          1. Assert: _toIndex_ + _count_ &le; _toSize_.
-          1. Repeat, while _count_ &gt; 0,
+          1. Assert: _fromIndex_, _toIndex_, and _count_ are Numbers whose mathematical value is an integer &ge; 0<sub>𝔽</sub>.
```

Comment:
Subscript here seems wrong, since we mentioned `Numbers whose mathematical value is an integer`.